.github/workflows/benchmark_signed_integer.yml

```diff
@@ -53,6 +62,27 @@ jobs:
         run: |
           echo "op_flavor=${{ toJSON(env.OP_FLAVOR) }}" >> "${GITHUB_OUTPUT}"
 
+      - name: Default benchmark type
+        if: github.event_name != 'workflow_dispatch'
```

Comment:
this means we don't get throughput on scheduled benchmarks right ?

is there a way in Grafana to not fetch benchmarks that don't match ? i.e. if I have old throughput data I don't want them to show next to the latest latency data, this way benchmarks are always consistent and we can tell when data is missing ?

Reply:
Regarding the data display, it's rather easy to show one or the other using SQL query since `::throughput::` is contained in the bench ID for any throughput benchmark.

About the scheduling, I can make latency + throughput the default measurements. Is that what we want ?
To be cost effective, we could do it only on the quarterly scheduled benchmarks.

Reply:
thing is when we look at both data, or when the blockchain will query the DB, how do we make sure the data they get is consistent ?

Reply:
We can make a top-down menu selector to switch between latency and throughput.

Reply:
This is implemented in CPU workflows.